### PR TITLE
add label to service, name to port, to make it selectable via Service Monitors

### PR DESCRIPTION
**Pull request by noyoshi**
Hi! I am trying to monitor my installation of volcano with ServiceMonitors. It looks like since the service is not labeled, we are not able to select it via the ServiceMonitor. 

I also added a named port, since I believe that is required. 

Here are the docs around the ServiceMonitor. 

https://docs.openshift.com/container-platform/4.10/rest_api/monitoring_apis/servicemonitor-monitoring-coreos-com-v1.html

Thanks!

### CODE REVIEW

**Comment by wangyang0616** [comment]
Welcome to the Volcano community.

You can execute the `make update-development-yaml` command to synchronize the update of `installer/helm/chart/volcano/templates/scheduler.yaml` to the `installer/volcano-development.yaml` file, which can ensure `helm` It is consistent with the deployment information of `yaml`, thanks.

**Comment by hwdef** [comment]
can you use podmonitor?

**Comment by noyoshi** [comment]
@hwdef I wasn't able to use a pod monitor since the pod doesn't have ports defined.

**Comment by hwdef** [comment]
> @hwdef I wasn't able to use a pod monitor since the pod doesn't have ports defined.

Please follow the tips above to modify your pr and fix DCO CI

**Comment by noyoshi** [comment]
@hwdef done! 
@wangyang0616 done!

Thanks for the help :)

**Comment by wangyang0616** [comment]
Please help to trigger CI in the background, thanks. 
cc @william-wang

**Review by hwdef — COMMENTED** [review]
please trigger the CI to run

**Comment by noyoshi** [comment]
Is triggering the CI something that I should be doing :) Thanks!

**Comment by wangyang0616** [comment]
> Is triggering the CI something that I should be doing :) Thanks!

The first PR in the community needs the maintainer to help trigger the operation of CI. 

Can you help trigger CI, Thanks! @william-wang

**Review by hwdef — COMMENTED** [review]
/lgtm

**Comment by noyoshi** [comment]
@hwdef I updated off master, so I think you might need to add your approval again to re-trigger CI. Thanks!

**Comment by noyoshi** [comment]
Ah I see, removed the merge commit

**Comment by hwdef** [comment]
@william-wang please retrigger the CI.

**Comment by noyoshi** [comment]
Would love to get this in if possible :) Thanks! @william-wang

**Review by shinytang6 — APPROVED** [review]

**Comment by noyoshi** [comment]
Do we still need someone to trigger ci?

**Comment by hwdef** [comment]
> Do we still need someone to trigger ci?

yes, @william-wang please trigger the CI.

**Review by william-wang — APPROVED** [review]
/lgtm

**Pull request merged**
